sound/soc/sof/core.c

```diff
@@ -197,7 +197,13 @@ int snd_sof_create_page_table(struct snd_sof_dev *sdev,
 		dmab->area, size, pages);
 
 	for (i = 0; i < pages; i++) {
-		u32 idx = (((i << 2) + i)) >> 1;
+		/* The number of valid address bits for each page is 20.
```

Comment:
nit pick, the recommended multi-line comment style is
/* 
 * The number...
 * line 2+
 */